WeatherService.Tests/BotServiceFactoryTesting/BotServiceFactoryShould.cs

```diff
@@ -0,0 +1,62 @@
+﻿using FluentAssertions;
+using WeatherService.Bots.Factories;
+using WeatherService.Bots.Interfaces;
+using WeatherService.Bots.RainBot;
+using WeatherService.Bots.SnowBot;
+using WeatherService.Bots.SunBot;
+using WeatherService.Utilities;
+
+namespace WeatherService.Tests.BotServiceFactoryTesting;
+public class BotServiceFactoryShould
+{
+    private IWeatherBotService _weatherBotService;
+
+    [Theory]
```

Comment:
Using [Theory] for a single case adds no benefit. Could you combine related cases into one test to fully leverage [Theory]?